subsys/suit/provisioning/soc/Kconfig.nrf9280

```diff
@@ -0,0 +1,167 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+#
+# WARNING:
+#  All addresses and sizes in this file must be sunchronized with the SUIT storage address
+#  defined in the Device Tree.
+#  Any modification of those values require alignment in the SDFW.
+#  Contents of the MPI (vendor name, class name, policies) may be modified by the
+#  product vendor.
```

Comment:
get the values from devicetree then? Why are there kconfigs duplicating what already exists